README.md

```diff
@@ -1,22 +1,14 @@
-# SSM BATS REST API service
-
-This is a Spring Boot REST Web Service for [BATS](https://github.com/jayjaybillings/bats)(Basic Artifact Tracking System).
-
-
-| Branch | Build | Coverage |
-|--------|-------|----------|
-| DEV    | [![DEV](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/badges/dev/pipeline.svg)](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/-/pipelines/dev/latest) | [![coverage report](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/badges/dev/coverage.svg)](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/-/commits/dev) |
-| QA    | [![DEV](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/badges/qa/pipeline.svg)](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/-/pipelines/qa/latest) | [![coverage report](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/badges/qa/coverage.svg)](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/-/commits/qa) |
-| PROD    | [![DEV](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/badges/main/pipeline.svg)](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/-/pipelines/main/latest) | [![coverage report](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/badges/qa/coverage.svg)](https://code.ornl.gov/rse/datastreams/ssm/backend/ssm-bats-rest-api/-/commits/qa) |
-
+# SSM Catalog Service
 
+This service is the data catalog via a REST API.
 
 ## Getting Started
 
+First, you need to [login to the GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
+
 The fastest way to spin up a working setup is to use docker-compose top-level file:
 
 ```
-docker login code.ornl.gov:4567
 docker-compose up
```

Comment:
Just wondering, I know the newer version of compose (v2 I think) uses the space instead of hyphen. It might be worth mentioning here?

Reply:
Done! Sorry for delay. Tested locally and working for me

Reply:
I don't think the compose file needs to be modified at all, it probably is as simple as removing the hyphen _and_ making sure that you have the compose plugin installed.

There could be something similar happening with Docker buildx replacing the old `build` command, which is currently deprecated.

Reply:
Commit that will fix it: https://github.com/smart-spectral-matching/ssm-service-catalog/commit/7a6abce35959d39d36f9a0604dca9fb4a6ecce68